fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeMetadata.java

```diff
@@ -228,6 +228,8 @@ private void collectDeltaLakePlanFiles(PredicateSearchKey key, Table table, Scal
 
         List<FileScanTask> files = Lists.newArrayList();
 
+        long estimateRowSize = table.getColumns().stream().mapToInt(column -> column.getType().getTypeSize()).sum();
```

Comment:
how can we estimate string columns more accurately

Reply:
It is unlikely that a more precise estimate can be made at present.